frontend/packages/console-dynamic-plugin-sdk/src/extensions/resource-metadata.ts

```diff
@@ -3,7 +3,7 @@ import { Extension, ExtensionDeclaration } from '../types';
 
 /** Customize the display of models by overriding values retrieved and generated through API discovery. */
 export type ModelMetadata = ExtensionDeclaration<
-  'console.resource-metadata',
+  'console.model-metadata',
```

Comment:
renaming an extension id is API breaking change

Reply:
@christianvogt asked for me IIRC, wdyt?

Reply:
If we make the change, we'll need to deprecate the old value and support both for a while. (Normal policy is 12 months / 3 releases.)

Reply:
Assuming it's an API we've already shipped.

Reply:
My original extension had identifier `console.resource-metadata` but I think I referred to it by name as 'model metadata'.

Since there is no implementation which consumes this extension I think it's fair to make any change now.

Let's just make sure we agree on the name.
I'm indifferent. Both are suitable to me.

Reply:
The API was shipped in 4.8 https://github.com/openshift/console/blob/release-4.8/frontend/packages/console-dynamic-plugin-sdk/src/extensions/resource-metadata.ts

Reply:
while it was shipped, there was no support for the extension

Reply:
If the extension was there, but did nothing, we can rename it. It shouldn't break any existing plugins.